### DISCUSSION TITLE
UMD does not work when quasar.umd.prod.js is stored locally

### DISCUSSION THREAD

**Discussion by JTrotta** [quasarframework/quasar#14764] [General - Components / Directives / etc]
Hello.
I use quasar UMD with no problem in an asp.net big project. I usually add quasar script canonically:
```
 <script src="https://cdn.jsdelivr.net/npm/vue@3/dist/vue.global.prod.js"></script>
 <script src="https://cdn.jsdelivr.net/npm/quasar@2.10.1/dist/quasar.umd.prod.js"></script>
```
And it works perfectly.
But I need to use local copy of script, than I change to:
```
 <script src="~/vue/dist/vue.global.prod.js"></script>
 <script src="~/quasar/dist/quasar.umd.prod.js"></script>
```
And even if both script are loaded VUE does not work. 
If I change vue script to NON prod version, it works:
```
 <script src="~/vue/dist/vue.global.js"></script>
 <script src="~/quasar/dist/quasar.umd.prod.js"></script>
```
WHY?

**Comment by MilosPaunovic** [comment]
This seems like an issue with Vue, not Quasar, so you might want to consider posting this on their repo.

**Reply by JTrotta** [reply]
Because of the strange behaviour, I suppose somebody else had the same problem here.
